src/EPR.Accreditation.Facade.Common/RESTservices/HttpAccreditationService.cs

```diff
@@ -108,5 +109,16 @@ private string GetAdressAsSingleLine(Dtos.Site dto)
 
             return address;
         }
+
+        public async Task<HasOverseasAgentDto> GetHasOverseasAgent(Guid accreditationExternalId)
```

Comment:
This layer should really be concerned with performing the neccessary calls to the API. Retrieving the specific property should be the concern of the service layer that you use. Therefore this call should just return the Accreditation object, and the service layer should get the "HasOverseasAgent" property and return that to the controller layer